com.unity.multiplayer.mlapi/Runtime/Core/RPCQueue/RPCQueueProcessing.cs

```diff
@@ -17,34 +14,26 @@ namespace MLAPI
     /// Inbound to invocation
     /// Outbound to send
     /// </summary>
-    internal class RPCQueueProcessing
+    internal class RpcQueueProcessing
     {
+
 #if DEVELOPMENT_BUILD || UNITY_EDITOR
         static ProfilerMarker s_MLAPIRPCQueueProcess = new ProfilerMarker("MLAPIRPCQueueProcess");
         static ProfilerMarker s_MLAPIRPCQueueSend = new ProfilerMarker("MLAPIRPCQueueSend");
 #endif
         // Batcher object used to manage the RPC batching on the send side
-        private MessageBatcher batcher = new MessageBatcher();
-        private int BatchThreshold = 1000;
+        private MessageBatcher m_batcher = new MessageBatcher();
+        private int m_BatchThreshold = 10;
```

Comment:
is this ever going to change at runtime? if not I'd suggest `const`.

Reply:
This will be resolved by Jeffrey's PR for rpc batching.